docs/Targets.md

```diff
@@ -199,7 +200,114 @@ arm-none-eabi-gdb
 add-symbol-file test-app/image.elf
 mon reset init
 ```
+## STM32U5
 
+### Scenario 1: TrustZone Enabled
+
+#### Example Description
+
+The implementation shows how to switch from secure application to non-secure application.
+Thanks to the system isolation performed to split the internal Flash and internal SRAM memories
+into two halves:
```

Comment:
This sentence is awkward.  Perhaps you want to say: 
Thanks to the system isolation, you can  split the internal Flash and internal SRAM memories
into two halves: